docs/pages/application-access/jwt/grafana.mdx

```diff
@@ -0,0 +1,116 @@
+---
+
+This guide will help you configure Grafana [JWT
+authentication](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/jwt/)
+with Teleport Application Access.
+
+## Prerequisites
+
+- Teleport cluster version >= `5.0.0` with running Auth/Proxy Services or
+  Teleport Cloud account.
+- Running [Application Service](../guides/connecting-apps.mdx).
+- Grafana cluster version >= `8.4.0`.
+
+## Step 1/3. Enable a JWT authentication in Grafana
+
+Update your Grafana configuration file to enable JWT authentication. This file
+is typically located at `/usr/share/grafana/conf/defaults.ini`.
+
+```toml
+#################################### Auth JWT ##########################
+[auth.jwt]
+enabled = true
+enable_login_token = false
+header_name = Teleport-Jwt-Assertion
+email_claim =
+username_claim = username
+jwk_set_url = https://proxy.example.com:3080/.well-known/jwks.json
+jwk_set_file =
+cache_ttl = 60m
+expect_claims = {}
+key_file =
+role_attribute_path =
+role_attribute_strict = false
+auto_sign_up = true
+url_login = false
+allow_assign_grafana_admin = false
+```
+
+Let's take a closer look at the parameters and their values:
+
+- Set `enabled` to `true` to enable JWT authentication.
+- Set `header_name` to the name of the header within which the JWT assertion is
+  sent.
+- Set `username_claim` to the claim within the JWT that contains the username.
+- Set `jwk_set_url` to the address where the JSON Web Key Set (JWKS) are
+  available. Replace `proxy.example.com:3080` with the host:port where the
+  Teleport Proxy is available.
+- Set `auto_sign_up` to `true` to allow automatic creation of users upon first
+  login via Teleport.
+
+<Admonition type="tip">
+If you are not ready to connect your production Grafana cluster to Teleport,
+you can use the below command to to launch an instance of Grafana configured
+with JWT authentication in Docker to connect to Teleport and experiment with.
+
+```code
+$ docker run \
+  -e GF_AUTH_JWT_ENABLED=true \
+  -e GF_AUTH_JWT_HEADER_NAME=Teleport-Jwt-Assertion \
+  -e GF_AUTH_JWT_USERNAME_CLAIM=username \
+  -e GF_AUTH_JWT_JWK_SET_URL=https://proxy.example.com:3080/.well-known/jwks.json \
+  -e GF_AUTH_JWT_AUTO_SIGN_UP=true \
+  -p 3000:3000 grafana/grafana-oss
+```
+
+If your proxy is using self-signed certificates, you will also need to add your
+self-signed public key to `/etc/ssl/certs/ca-certificates.crt` within the
+container.
+</Admonition>
+
+## Step 2/3. Register with Grafana in Teleport
+
+In your Teleport App Service configuration file, `teleport.yaml`, register an
+entry for Grafana:
+
+```yaml
+app_service:
+  enabled: "yes"
+  apps:
+  - name: "grafana"
+    uri: "https://grafana.example.com:3000"
+```
+
+Replace `https://grafana.example.com:3000` with the URL of your Grafana
+instance.
+
+<Admonition type="tip">
+You can also use [dynamic registration](../guides/dynamic-registration.mdx).
```

Comment:
We're trying to avoid linking users outside of how-to guides unless it's necessary in order to help users get through our guides quickly—would it be possible to include an example of the Grafana `app` resource you would create instead of this link?

If we require a running Application Service instance, this might also be a good candidate for a `TabItem` in a `Tabs` component where a second `TabItem` includes the configuration change above.